.yarnrc.yml

```diff
@@ -11,13 +11,9 @@ initFields:
   publishConfig:
     provenance: true
 
+nodeLinker: node-modules
```

Comment:
@cheempz This would be the line, the main advantage is that it's the Node ecosystem default which has better tooling support and makes more sense to be testing against

Reply:
Thanks, that makes sense.